homeassistant/components/stookwijzer/const.py

```diff
@@ -10,6 +10,6 @@
 class StookwijzerState(StrEnum):
     """Stookwijzer states for sensor entity."""
 
-    BLUE = "blauw"
-    ORANGE = "oranje"
-    RED = "rood"
+    CODEYELLOW = "codeYellow"
+    CODEORANGE = "codeOrange"
+    CODERED = "codeRed"
```

Comment:
```suggestion
    CODE_YELLOW = "code_yellow"
    CODE_ORANGE = "code_orange"
    CODE_RED = "code_red"
```

Reply:
Also, please update the PR description to mention that this is a breaking change, since we're updating the possible states

Reply:
Thanks, done already.

Reply:
> codeYellow

As I think about it, this will break the integration. The codeYellow is also a format used on the site, so I used it in the API as well...